leveller.js

```diff
@@ -61,16 +61,24 @@
     var diff = targetHeight - currentHeight;
     if (diff === 0) return;
 
+    var isHeightProperty = (this.options.cssProperty.indexOf('eight') > 0);
     var $styleElement = this.options.cssSelector ? $element.find(this.options.cssSelector) : $element;
-    var styleValue = targetHeight;
-    if (this.options.cssProperty.indexOf('eight') < 0) {
+    var styleValue;
+
+    if (isHeightProperty && this.options.cssSelector) {
+      styleValue = $styleElement[this.options.heightMethod]() + diff;
```

Comment:
Yes, checking for its existence. If the `cssSelector` option is anything truthy that jQuery would accept as a selector, I have to treat the height property as additive.